### PR TITLE
feat(escalating): Update alert design for escalating issues

### LINKED ISSUE #48756: Show status for email alerts
See the “Add Status” here: https://www.figma.com/file/BeUa34s0rfYAPWx0fY3JmG/Emails?node-id=301-527&t=vfEvmbIKPXCykUX6-0 

We should show the status clearly in the emails

### PR DESCRIPTION
**Pull request by scttcper**
- Changes the title to include the inbox reason
- Adds a table with the project, environment, level

fixes #48756

before:
Screenshot 2023-05-08 at 3 20 43 PM

after (ignore the event id swapped for short id)
Screenshot 2023-05-08 at 3 20 12 PM

### CODE REVIEW

**Comment by scttcper** [comment]
@lobsterkatie the culprit is only present 50% of the time in the debug template. It should still be there.
I think the text is cut a specific limit because email css is pretty limited https://www.caniemail.com/features/css-text-overflow/

**Comment by scttcper** [comment]
I guess we could always render the culprit https://github.com/getsentry/sentry/pull/48762/files#diff-cfcb0a09405498fee62cfebea4073c1478110f06c93f464aa98d9e7ee25e0bdbR478

**Comment by lobsterkatie** [comment]
> I think the text is cut to a specific limit because email css is pretty limited [caniemail.com/features/css-text-overflow](https://www.caniemail.com/features/css-text-overflow/) and its saving room for the culprit which is usually there.

So in the new design, the culprit would go next to the title rather than underneath it? (I ask because in the old design it was underneath and yet the title was still cut off.)

**Comment by lobsterkatie** [comment]
> how the error title/mesage is displayed is unchanged.

Ohhh, I think I get what the confusion is. In the screenshot in the PR description, the culprit is so long that instead of showing up next to the issue title, it shows up under it, so I was assuming it just always lived down there. In your second set of screenshots, it's a shorter culprit, so it fits on the same line.

File: src/sentry/models/groupinbox.py [hunk 1]
```diff
@@ -153,3 +153,14 @@ def get_inbox_details(group_list):
...
     return inbox_stats
+
+
+def get_inbox_reason_text(inbox_reason: GroupInboxReason):
+    if inbox_reason == GroupInboxReason.NEW:
+        return "New issue"
+    elif inbox_reason == GroupInboxReason.REGRESSION:
+        return "Issue regressed"
+    elif inbox_reason == GroupInboxReason.ONGOING:
+        return "Issue ongoing"
```
Comment (armenzg) [inline src/sentry/models/groupinbox.py:164]: Would "Regressed issue" and "Ongoing issue" work better? ("skinned rabbit" vs "rabbit skinned") I always have trouble in my head to figure out what is the right way.

File: src/sentry/notifications/notifications/rules.py [hunk 2]
```diff
@@ -117,10 +118,17 @@ def get_context(self) -> MutableMapping[str, Any]:
             fallthrough_choice=self.fallthrough_choice,
         )
         fallback_params: MutableMapping[str, str] = {}
+        group_inbox_reason = GroupInbox.objects.filter(
+            group=self.group, project=self.project
+        ).first()
+        group_header = "New Alert"
+        if group_inbox_reason:
+            group_header = get_inbox_reason_text(GroupInboxReason(group_inbox_reason.reason))
```
Comment (armenzg) [inline src/sentry/notifications/notifications/rules.py:126]: nit: I would pass `group_inbox_reason` directly to `get_inbox_reason_text` and let it handle the case where is not defined within it, thus, you would give all responsibility to the function.

**Review by armenzg — APPROVED** [review]
👍🏻  Only minor optional nit.

**Pull request merged**